vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -38,7 +46,20 @@
      */
     @Override
     public ExhibitionAboutPage store(ExhibitionAboutPage exhibitionAboutPage) {
+        IExhibition exhibition = exhibitionManager.getStartExhibition();
+        exhibition.setAboutPageConfigured(true);
+        exhibitionManager.storeExhibition((Exhibition)exhibition);
```

Comment:
the above two lines only need to happen if `setAboutPageConfigured` returns false, right?